content/game_process.tex

```diff
@@ -149,8 +149,8 @@ \subsection{Kick-off}
 For kick-off, the robots listening wirelessly to the GameController, run through three states: \emph{ready}, \emph{set}, and \emph{playing}.
 It is to a team's responsibility to have their robots listening to the GameController!
 
-In the ready state, the robots must walk to a legal kick-off positions for their team.
-The attacking team can be positioned anywhere within their own half including center circle. The robot executing the kick-off is also allowed to be positioned inside the whole center circle without blocking its center.
+In \textit{Ready} state, the robots must walk to a legal kick-off positions for their team.
+The kick-off team can be positioned anywhere within their own half including their half of the center circle. Only one robot of the kick-off team is allowed to stay within the whole center circle during \textit{Set} until the ball is in play. All robots except the closest to the center point of the center circle in the opponent's half of the center circle get penalized by ``Illegal Positioning''~(\cf \cref{sec:illegal_positioning}). This holds for robots who are positioned there wrongly in \textit{Set} or entering the opponent's half before the ball is in play. If the kick-off team has a robot positioned in the opponent's half of the center circle, only this robot, called kick-off player, is allowed to execute a kick-off. If another robot of the kick-off team is executing the kick-off than the kick-off player, this robot gets penalized with ``Illegal Positioning'' and a moved ball gets moved back. Even when the ball is free only all defending robots and the kick-off player are allowed to kick the ball. This restriction ends when the game state switches to \textit{Playing}.
```

Comment:
All robots **in the opponent's half of the center circle,** except the closest to the center point of the center circle**,** get penalized by ``Illegal Positioning''~(\cf \cref{sec:illegal_positioning}).